drv/gimlet-seq-api/src/lib.rs

```diff
@@ -28,6 +27,15 @@ pub enum SeqError {
     ServerRestarted,
 }
 
+/// Errors returned by `read_fpga_regs`.
+#[derive(Copy, Clone, Debug, FromPrimitive, Eq, PartialEq, IdolError)]
+pub enum ReadFpgaRegsError {
+    Failed = 1,
```

Comment:
Maybe add a comment noting that this variant is _probably_ illusory because of the SPI error API issue, with a TODO to revisit it.

Reply:
will do!